lib/module_importer.js

```diff
@@ -63,8 +63,9 @@ function makeImporter(eyeglass, sass, options, fallbackImporter) {
     var relativePath = fragments.slice(1).join("/");
     var pkgRootDir = isRealFile ? importUtils.packageRootDir(path.dirname(prev)) : root;
     var jsFile = importUtils.getModuleByName(moduleName, pkgRootDir);
-    // includePaths here is a string, despite the config option being an Array
-    var includePaths = options.includePaths.split(path.delimiter);
+    // in some cases, includePaths here is a string, despite the config option being an Array
+    // other cases, it is an Array (or undefined if it was not passed as an option)
+    var includePaths = (options.includePaths && options.includePaths.split) ? options.includePaths.split(path.delimiter) : options.includePaths || [];;
```

Comment:
Thanks for fixing this.

One suggestion: can we move this logic into a function:

```js
function normalizeIncludePaths(paths) {
  paths = paths || [];

  // if it's string like...
  if (paths.split) {
    // split it into an array
    paths = paths.split(paths.delimiter);
  }
  
  // ensure that we're always using an array
  if (!Array.isArray(paths)) {
    console.warn("includePaths should be an array but got " + typeof paths);
    return [];
  }

  return paths;
}

...

var includePaths = normalizeIncludePaths(options.includePaths);
```

it's a bit more verbose but there's a clear abstraction for doing this normalization